cmd/node/node.go

```diff
@@ -89,6 +90,9 @@ func main() {
 			flag.Usage()
 			panic("forwarding-target set when sequencer enabled")
 		}
+		if nodeConfig.Node.EnableL1Reader && nodeConfig.Node.InboxReader.HardReorg {
+			panic("hard reorgs cannot safely be enabled with sequencer mode enabled")
```

Comment:
this test should probably go to createNodeImpl

Reply:
It was originally there but the TestSequencerInbox requires hard reorgs